kivy/base.py

```diff
@@ -615,3 +615,15 @@ def stopTouchApp():
         return
     Logger.info('Base: Leaving application in progress...')
     EventLoop.close()
+
+
+import os
+os.environ['KIVY_WINDOW'] = 'none'
+from kivy.logger import Logger
+from kivy.base import EventLoop
+Logger.info('Starting test')
+try:
+    EventLoop.ensure_window()
+except SystemExit as e:
+    Logger.info(f'SystemExit: {e}')
+    print(f'SystemExit: {e}')
```

Comment:
Hi @Aakiboi,

Can you help me understand why this is needed for your use-case?